lib/logstash/filters/elasticsearch.rb

```diff
@@ -478,6 +479,15 @@ def test_connection!
     end
   end
 
+  def test_serverless_connection!
+    begin
+      get_client.client.info(:headers => LogStash::Filters::ElasticsearchClient::DEFAULT_EAV_HEADER ) if get_client.serverless?
```

Comment:
use `info` instead of `ping` because the error message from ping is insufficient